cypress-tests/cypress/e2e/admin/pageBuilder/templates/templatePreview.cy.ts

```diff
@@ -0,0 +1,64 @@
+import { customAlphabet } from "nanoid";
+
+context("Page Builder - Blocks", () => {
+    const nanoid = customAlphabet("abcdefghijklmnopqrstuvwxyz");
+    const pageTemplateData1 = {
+        title: nanoid(6),
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+    const pageTemplateData2 = {
+        title: nanoid(6),
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+    const pageTemplateData3 = {
+        title: nanoid(6),
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllTemplates();
+        cy.pbCreatePageTemplate(pageTemplateData1);
+        cy.pbCreatePageTemplate(pageTemplateData2);
+        cy.pbCreatePageTemplate(pageTemplateData3);
+    });
+
+    it("Should be able to create a page and view all existing templates in it", () => {
+        cy.visit("/page-builder/pages?folderId=root");
+        cy.wait(1000);
+        cy.findByTestId("new-page-button").click();
+        cy.wait(500);
+        cy.contains("Pick a template for your new page").should("exist");
+        cy.contains(pageTemplateData1.title).should("exist");
+        cy.contains(pageTemplateData1.description).should("exist");
+        cy.contains(pageTemplateData2.title).should("exist");
+        cy.contains(pageTemplateData2.description).should("exist");
+        cy.contains(pageTemplateData3.title).should("exist");
+        cy.contains(pageTemplateData3.description).should("exist");
+
+        cy.get(".css-1rl9ll7-listStyle .css-5bicyh-listItem").each(item => {
```

Comment:
Can we have a test ID for this? Please attach a sshot and I'll see if it can be done.

Reply:
![image](https://github.com/webiny/webiny-js/assets/35311113/3a2db3b6-eb6d-4f60-b631-558b7f386e3a)

Reply:
`pb-pages-list-use-blank-template-btn`
![image](https://github.com/webiny/webiny-js/assets/5121148/dbdb1046-391c-4ee6-9435-4f69309049bb)